nautobot/core/utils/utils.py

```diff
@@ -27,8 +27,8 @@
 from django_filters.utils import verbose_lookup_expr
```

Comment:
I don't like having `nautobot.core.utils.utils` - would appreciate suggestions as to a better name for this submodule.

Reply:
Possibly `utils.functions`? Or possibly split this into several more narrowly targeted modules, such as `utils.lookup` (`get_route_for_model`, `get_related_class_for_model`, etc.)?

Reply:
> Could we eliminate the folder and just have `utils.py`?

Are you thinking to merge all of these files into a single `utils.py`? To my mind that's not a great solution since it results in that one file being a massive dumping-ground for miscellaneous functionality:

```
> ls -1 nautobot/core/utils
__init__.py
config.py
deprecation.py
git.py
logging.py
ordering.py
paginator.py
permissions.py
query_functions.py
utils.py
```

Reply:
ACK

How big is too big to be broken up into smaller files? < 500?

Reply:
See [comment above](https://github.com/nautobot/nautobot/pull/3132#discussion_r1085963290) for my attempt at getting rid of `nautobot.core.utils.utils` by splitting it out into other modules.